pkg/controller/common/rollout/rollout_webhook.go

```diff
@@ -125,7 +124,7 @@ func callWebhook(ctx context.Context, resource klog.KMetadata, phase string, rw
 	}
 	if !accepted {
 		err := fmt.Errorf("http request to the webhook not accepeted, http status = %d", status)
-		klog.V(common.LogDebug).InfoS("the status is not expected", "expected status", rw.ExpectedStatus)
+		klog.InfoS("The status is not expected", "expected status", rw.ExpectedStatus, "err", err)
```

Comment:
Maybe `klog.ErrorS(err, "msg","k","v")` is better for logging error messages?

Reply:
yeah, It seems better